client/src/app/components/modify-members/modify-members.component.html

```diff
@@ -1,12 +1,3 @@
 <p>Inside {{view}} Component</p>
 <button (click)="obtainClients()">Obtain Clients</button>
-<ul *ngFor="let pers of users">
-    <li>
-        username = {{pers.user_name}},
-        firstname = {{pers.first_name}},
-        lastname = {{pers.last_name}}, 
-        role = {{pers.role}},
-        is_active = {{pers.is_active}},
-        e-mail = {{pers.email}}
-    </li>
-</ul>
\ No newline at end of file
+<app-client-gridview></app-client-gridview>
```

Comment:
So here you are just injecting the table client-gridview. Now what I'm suggesting is inside the client-gridview component you create 2 extra columns for the table and add the update and delete buttons that will be attached to the id of the particular user in that row. Also those 2 column will only be displayed if the logged-in user has the particular clearance and this can be handled using a ngIf*.

Reply:
This does not refer to the current PR. The goal of this PR was simply to obtain the already gotten clients and make a simple table out of it with addition to @NicholasNagy 's request of adding a search feature. I do not know if this is the smartest way or if we should go with an input text boxes along with the previous data as a placeholder. These are different ways and I believe this should be brought to it's own issue, it's own PR at the very least and depends on the mockups which have not yet been created for this